src/shared/components/blocks/subscribe/Subscribe.module.css

```diff
@@ -1,3 +1,149 @@
+@import "../../../styles/colors";
+@import "../../../styles/custom-medias";
+@import "../../../styles/mixins/typography";
+
 .subscribe {
-    background: rosybrown;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--color-celtic);
+}
+
+.joinUs {
+    position: relative;
+    flex: 1;
+    text-align: center;
+}
+
+.firstText {
+    @mixin typography-h1 rem;
+    color: var(--color-tan);
+}
+
+.secondText {
+    width: 408px;
+    @mixin typography-body-1 rem;
+    margin: 70px auto 0 auto;
+    color: var(--color-white);
+    font-size: 18px;
+}
+
+.image {
+    height: 100%;
+    flex: 1;
+    color: var(--color-tan);
+
+    & div svg {
+        width: 75% !important;
+        height: 75% !important;
+    }
+}
+
+form {
```

Comment:
you need to put a class before this since the tag doesn't be hashed like the classes.